gateway-home/modules/ROOT/pages/gateway-overview.adoc

```diff
@@ -6,167 +6,48 @@ endif::[]
-* Add any other complex capability to an API without having to write any code.
+Mule Runtime includes an embedded Mule Gateway. Using this gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
 
-Mule Gateway protects a single Mule API. The key advantage is that for application developers building Mule applications, basic endpoint protection becomes trivial.
+Overall, Mule Gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
 
-You can also configure Mule Runtime in Anypoint Runtime Manager as a CloudHub Proxy application, protecting multiple backends. However, in this case the gateway cannot be easily scaled to support high-availability and high-performance applications. Further, Mule Runtime consumes more computes or vCores than the smaller-footprint Flex Gateway.
+If you want to apply policies and generate analytics info, you must use either:
 
-Building custom policies on the Mule gateway is akin to building an application with Java using the Mule DSL. Building a custom policy in Flex Gateway is based on Envoy-provided Rust WASM SDKs. A policy written for Mule Gateway cannot be reused in Flex Gateway and vice versa, as the underlying architectures are fundamentally different.
+* An API defined in a Mule application.
+* A Mule application that proxies your existing backend implementation.
 
 For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[Mule Gateway Capabilities].
 
-MuleSoft recommends you choose Flex Gateway for high-availability and high-performance applications.
-
-[%header%autowidth.spread,cols="a,a,a"]
-|===
-| | Flex Gateway | Mule Gateway
-
-| *Summary*
-| Envoy-based API gateway to secure all APIs, Mule and non-Mule, running anywhere.
-| Java-based API gateway embedded into the Mule Runtime.
-
-| *Tech Stack*
-| Underlying engine build upon Envoy, and FluentBit for logging.
-| Java Spring Application embedded into the Mule Runtime.
-
-| *Use Case*
-| High-performant and high-availability. Secure any API anywhere.
-| Secure a Mule API "Basic Endpoint", or enable a dedicated proxy as an embedded library in a Mule Runtime. Also available as a Mule proxy application in CloudHub.
-
-| *Key Capabilities*
-| Small footprint. Multiple deployment patterns and modes, including as a native Kubernetes Ingress controller. Connected Mode as well as Local Mode available.
-| Same technology as Mule integration applications. API auto-discovery for Mule applications. Connected Mode only.
-
-| *Custom Policy Development*
-| Rust WASM SDKs.
-| Java and Mule DSL.
-
-|===
+== Anypoint Service Mesh
```

Comment:
Why is this here? How does it relate to the two gateways? 

Maybe instead of including this content here, mention it and link to the Service Mesh docs?

Reply:
Actually, I got confirmation that Service Mesh is defunded. PMs are requesting I remove mention of it.